prisma/migrations/20230601181102_create_table_schedule_topics/migration.sql

```diff
@@ -0,0 +1,28 @@
+/*
+  Warnings:
+
+  - Added the required column `schedule_topic_id` to the `schedule_monitoring` table without a default value. This is not possible if the table is not empty.
+
+*/
+-- AlterTable
+ALTER TABLE `schedule_monitoring` ADD COLUMN `schedule_topic_id` INTEGER NOT NULL;
```

Comment:
Essa coluna não pode estar com `NOT NULL` senão vai dar erro pra executar a migração, já que os agendamentos que já existem não vão ter um tópico relacionado e aí isso vai ferir essa restrição. A coluna tem que ser nullable

Reply:
boa